.circleci/config.yml

```diff
@@ -13,33 +13,39 @@ workflows:
           aws-secret-access-key: $OPS_AWS_SECRET_ACCESS_KEY
           account-url: $OPS_ECR_URL
           puller-account-ids: '["301258414863", "487088987264"]'
+          extra-docker-args: "--build-arg VERSION=$(git describe --tags)"
           repo: rdoc-r-worker
-          pre-build-steps:
-            - run:
-                name: Build application docker image
-                command: |
-                  docker login -p ${DOCKERHUB_PASSWORD} -u ${DOCKERHUB_USER} dockerhub.datacamp.com:443
```

Comment:
I think this is still needed. I added it because we are using one of our base images in the Dockerfile to logging to dockerhub is still required

Reply:
You're right!